src/reset.css

```diff
@@ -0,0 +1,11 @@
+body {
+  margin: 0;
+  width: 200px;
```

Comment:
you shouldn't add any width to the body, delete this property from here
you should just specify the width for the product card